packages/search-ui-app-search-connector/src/__tests__/requestAdapters.test.js

```diff
@@ -24,12 +24,14 @@ const request = {
     {
       field: "initial",
       values: ["values"],
-      type: "all"
+      outerType: "all",
```

Comment:
Same thing here, don't update this test, create new cases (Or even just new objects within this test case) to test `outerType`.